WriteHDLUtils.py

```diff
@@ -401,7 +411,7 @@ def writeTemplatePars_TC(aTCModule):
                 mask = 4
             elif phi == 'D':
                 mask = 8
-            assert(mask > 0)
+            #assert(mask > 0)
```

Comment:
Why is the "assert" statement commented out? Doesn't it protect against unexpected cfgs?